src/strategies/npm-audit.js

```diff
@@ -9,10 +9,22 @@ import { standardizeVulnsPayload } from "./vuln-payload/standardize.js";
 export function NPMAuditStrategy() {
   return {
     strategy: VULN_MODE.NPM_AUDIT,
-    hydratePayloadDependencies
+    hydratePayloadDependencies,
+    getVulnerabilities
   };
 }
 
+async function getVulnerabilities(path = "") {
+  const arborist = new Arborist({ ...NPM_TOKEN, path });
+
+  try {
+    return (await arborist.audit()).toJSON().vulnerabilities;
+  }
+  catch {
+    return null;
```

Comment:
IMHO depending on the data structure we want to return in a happy path case, it might make sense to return something else than `null`, either an `empty array` or an `empty object`.

Reply:
Yes that make sens